lectures/greek_square.md

```diff
@@ -48,17 +58,37 @@ In this lecture, we'll describe this method.
 
 We'll also use invariant subspaces to describe variations on this method that are faster.
 
-## Primer on second order linear difference equation
+## Second order linear difference equations
+
+Before telling how the ancient Greeks computed square roots, we'll provide a quick introduction
+to second order linear difference equations.
 
-Consider the following second-order linear difference equation
+We'll study  the following second-order linear difference equation
 
 $$
 y_t = a_1 y_{t-1} + a_2 y_{t-2}, \quad t \geq 0
 $$ (eq:2diff1)
 
 where $(y_{-1},  y_{-2})$ is a pair of given initial conditions.  
 
-We want to find expressions for $y_t, t \geq 0$ as functions of the initial conditions  $(y_{-1},  y_{-2})$:
+Equation {eq}`eq:2diff1` is actually an infinite number of linear equations in the sequence
+$\{y_t\}_{t=0}^\infty$.
+
+There is one equation each for $t = 0, 1, 2, \ldots$.  
+
+We could follow an approach taken in this QuantEcon lecture {doc}`present values<pv>` and stack all of these equations into a single matrix equation that we would then solve by using matrix inversion.
+
+```{note} In the present instance, the matrix equation would multiply a countably infinite dimensional square matrix by  a countably infinite dimensional vector.  With some qualifications, matrix multiplication and inversion tools apply to such an equation.
```

Comment:
```suggestion
```{note} 
In the present instance, the matrix equation would multiply a countably infinite dimensional square matrix by  a countably infinite dimensional vector.  With some qualifications, matrix multiplication and inversion tools apply to such an equation.
```